langchain4j-vertex-ai-gemini/src/test/java/dev/langchain4j/model/vertexai/VertexAiGeminiChatModelIT.java

```diff
@@ -390,6 +408,23 @@ void should_handle_parallel_function_calls() {
 
         assertThat(inventoryStock).containsIgnoringCase("ABC123");
         assertThat(inventoryStock).containsIgnoringCase("XYZ789");
+
+        // when
+        allMessages.add(messageResponse.content());
+
+        allMessages.add(ToolExecutionResultMessage.toolExecutionResultMessage(null, "getProductInventory",
+            "{\"product_id\":\"ABC123\", \"stock\": 10}"));
+        allMessages.add(ToolExecutionResultMessage.toolExecutionResultMessage(null, "getProductInventory",
+            "{\"product_id\":\"XYZ789\", \"stock\": 5}"));
+
+        messageResponse = model.generate(allMessages, stockInventoryToolSpec);
+
+        System.out.println("Final response = " + messageResponse.content().text());
+
+        // then
+        String text = messageResponse.content().text();
+        List<String> sequence = Arrays.asList("ABC123", "more", "XYZ789");
+        assertThat(text.split(" ")).containsAll(sequence);
```

Comment:
you probably wanted to check that the order is correct?
```suggestion
        assertThat(text.split(" ")).containsSubsequence("ABC123", "more", "XYZ789");
```